### PR TITLE
[4920]account/email: adding template for account already exists case …

### LINKED ISSUE #4920: TemplateDoesNotExist: account/email/account_already_exists.de.email, account/email/account_already_exists.en.email
Sentry Issue: [MEINBERLIN-DEV-81](https://sentry.liqd.net/organizations/liqd/issues/2951/?referrer=github_integration)

```
TemplateDoesNotExist: account/email/account_already_exists.de.email, account/email/account_already_exists.en.email
(16 additional frame(s) were not displayed)
...
  File "meinberlin/apps/users/adapters.py", line 46, in send_mail
    return UserAccountEmail.send(email, template_name=template_prefix, **context)
  File "adhocracy4/emails/mixins.py", line 38, in send
    return cls().dispatch(object, *args, **kwargs)
  File "adhocracy4/emails/base.py", line 127, in dispatch
    (subject, text, html) = self.render(template, context)
  File "adhocracy4/emails/base.py", line 98, in render
    template = select_template(
  File "django/template/loader.py", line 47, in select_template
    raise TemplateDoesNotExist(', '.join(template_name_list), chain=chain)
```

### PR DESCRIPTION
**Pull request by khamui**
…and add test command

closes #4920

### CODE REVIEW

**Review by fuzzylogic2000 — CHANGES_REQUESTED** [review]
Did it work for you? I get
```
Internal Server Error: /accounts/signup/
Traceback (most recent call last):
  File "/home/katharina/a4-meinberlin/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/katharina/a4-meinberlin/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/katharina/a4-meinberlin/venv/lib/python3.10/site-packages/django/views/generic/base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/katharina/a4-meinberlin/venv/lib/python3.10/site-packages/django/utils/decorators.py", line 43, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/katharina/a4-meinberlin/venv/lib/python3.10/site-packages/allauth/decorators.py", line 20, in wrap
    resp = function(request, *args, **kwargs)
  File "/home/katharina/a4-meinberlin/venv/lib/python3.10/site-packages/django/utils/decorators.py", line 43, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/katharina/a4-meinberlin/venv/lib/python3.10/site-packages/django/views/decorators/debug.py", line 89, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "/home/katharina/a4-meinberlin/venv/lib/python3.10/site-packages/allauth/account/views.py", line 234, in dispatch
    return super(SignupView, self).dispatch(request, *args, **kwargs)
  File "/home/katharina/a4-meinberlin/venv/lib/python3.10/site-packages/allauth/account/views.py", line 77, in dispatch
    response = super(RedirectAuthenticatedUserMixin, self).dispatch(
  File "/home/katharina/a4-meinberlin/venv/lib/python3.10/site-packages/allauth/account/views.py", line 207, in dispatch
    return super(CloseableSignupMixin, self).dispatch(request, *args, **kwargs)
  File "/home/katharina/a4-meinberlin/venv/lib/python3.10/site-packages/django/views/generic/base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/katharina/a4-meinberlin/venv/lib/python3.10/site-packages/allauth/account/views.py", line 105, in post
    response = self.form_valid(form)
  File "/home/katharina/a4-meinberlin/venv/lib/python3.10/site-packages/allauth/account/views.py", line 248, in form_valid
    self.user = form.save(self.request)
  File "/home/katharina/a4-meinberlin/meinberlin/apps/users/forms.py", line 115, in save
    user.get_newsletters = self.cleaned_data["get_newsletters"]
AttributeError: 'NoneType' object has no attribute 'get_newsletters'
```
I hoped it would be as easy as adding the missing email, but seems it's not. :(

**Comment by khamui** [comment]
@fuzzylogic2000 @Rineee now it validates the field, but does not send the email. Do you have an idea what I miss here?

**Comment by fuzzylogic2000** [comment]
> @fuzzylogic2000 @Rineee now it validates the field, but does not send the email. Do you have an idea what I miss here?

@khamui Did you check the allauth code? Wher is that done and validated? I guess, it shold also sent that email in that error and not only raise the validationError? But I don't know and would also need to check the allauth code.

**Comment by khamui** [comment]
> > @fuzzylogic2000 @Rineee now it validates the field, but does not send the email. Do you have an idea what I miss here?
> 
> @khamui Did you check the allauth code? Wher is that done and validated? I guess, it shold also sent that email in that error and not only raise the validationError? But I don't know and would also need to check the allauth code.

@fuzzylogic2000 it seems that I cannot throw a Validation error in `save()` and in `clean_email` I do not have `request`. The last commit shows the validation error and sends the email. But the email does not contain a proper reset_password link, because there is no request...maybe we can talk about it, maybe you have an idea or a hint :)

File: meinberlin/apps/users/forms.py [hunk 1]
```diff
@@ -112,10 +110,11 @@ def __init__(self, *args, **kwargs):
 
     def save(self, request):
         user = super(TermsSignupForm, self).save(request)
-        user.get_newsletters = self.cleaned_data["get_newsletters"]
-        user.get_notifications = self.cleaned_data["get_notifications"]
-        user.save()
-        return user
+        if user:
```
Comment (khamui) [inline meinberlin/apps/users/forms.py:113]: the super().save() sends the email and returns None in case of email already exists.

**Review by fuzzylogic2000 — APPROVED** [review]
Perfect!

**Pull request merged**